.github/workflows/integration-tests-on-release.yaml

```diff
@@ -256,17 +256,12 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: setup latest cluster resources
-        if: ${{ matrix.kubernetes-versions }} != "v1.24.1"
+      - name: setup cluster resources
         uses: ./.github/workflows/integration-test-config-latest-k8s
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           kubernetes-version: ${{ matrix.kubernetes-versions }}
 
-      - name: setup older cluster resources
-        if: ${{ matrix.kubernetes-versions }} == "v1.19.16"
-        uses: ./.github/workflows/integration-test-config-older-k8s
-
```

Comment:
Really delete? I see that `v1.19.16` is still in the test matrix.

Reply:
This had been added with #10699 and was needed for v1.17. I had updated it in #10834 for this condition to be v1.19, but then it just caused more problems.